### DISCUSSION TITLE
[Bug] Build throwing  out of memory exception after adding Fluent-UI v9 components

### DISCUSSION THREAD

**Discussion by ShrutiJaiswal1494** [microsoft/fluentui#24645] [Q&A]
Hi Team,

Our App is currently using Fluent-UI V8 & V9 components (which we recently added to our project) to use some of the new controls (tab) there. After adding the V9 components our build is throwing out of memory exception when running on Azure instances. I have tried setting the `max_old_space_size` in the build command to different values but it didn't work. Any suggestions on how to fix this?

<img width="495" alt="image" src="https://user-images.githubusercontent.com/79670853/188074005-713c9ab2-2a1a-49c3-a80e-eb9bfaee6654.png">

**Comment by micahgodbolt** [comment]
I can't imagine this has anything to do with our controls. Have you tried removing our controls and seeing if the error goes away? Is it any control that causes the issue, or just one specific control? If you do isolate the issue, you should probably open up a bug.

**Comment by ShrutiJaiswal1494** [comment]
I also tried using the Button control instead of the tab but same error. After trying different combinations, I think it's the `FluentProvider` tag messing up the memory. Not sure if the styling is causing some conflicts here. Sharing the App start code in case that helps.

```
            ReactDOM.render(
                <FluentProvider theme={teamsLightTheme}>
                    <Provider store={configureStore()}>
                        <BrowserRouter>
                            <Switch>
                                <Route path="/" exact component={RootContainer} />
                                <Redirect to="/" />
                            </Switch>
                        </BrowserRouter>
                    </Provider>
                </FluentProvider>,
                rootElement
            );
```
